crawler.py

```diff
@@ -1,32 +1,43 @@
+import requests
+
 from bs4 import BeautifulSoup
 
-import requests
 
-def parse():
+def crawl():
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 '
+                      '(KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36'
+    }
+
+    games = []
 
-	HEADERS = {
-		'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36'
-	}
+    page_number = 1
+    while True:
+        page_url = f'https://marketplace.xbox.com/en-US/Games/Xbox360Games?Page={page_number}'
 
-	max_page = 43
-	pages = []
+        print(f"Loading page #{page_number}: {page_url}")
+        response = requests.get(page_url,headers=headers)
+        print("Done")
 
-	for x in range(1, max_page + 1):
```

Comment:
Two comments here:
1. In case this approach would be applicable, it's still a bit better to do `for x in range(0, max_page)` and then user `x + 1`.
2. ...but, taking to account that actual amount of page eventually can change (pages will be added or removed), it's better to implement code, which does not care how many pages there are. You just start crawling from page 1, and stop when some terminal condition is reached.